imports/pages/give/history/Filter.js

```diff
@@ -305,6 +306,7 @@ export default class Filter extends Component {
                   onClick={this.startClick}
                   active={this.state.customEndActive}
                   className={this.state.customDateDisabled && "tag--disabled"}
+                  icon={!this.state.customEndActive && <span className="icon-calendar push-half-left" style={{ position: "relative", top: "-2px" }} />}
```

Comment:
@richarddubay let's move the span here and above to outside the component. It's a slight performance hit to create an element in each render